openc3/python/openc3/conversions/processor_conversion.py

```diff
@@ -45,6 +45,7 @@ def __init__(
             self.converted_bit_size = int(converted_bit_size)
         if ConfigParser.handle_none(converted_array_size):
             self.converted_array_size = int(converted_array_size)
+        self.params = [self.processor_name, self.result_name]
```

Comment:
Missing converted_ params

Reply:
Base class records those ... they are output in the to_config method. as_json outputs them separately and the remaining parameters get put into params.